package.json

```diff
@@ -5,7 +5,8 @@
   "main": "app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "npx nodemon"
+    "start": "npx nodemon",
+    "dev": "nodemon app.js"
```

Comment:
I think line 8 and line 9 are basically the same. It will trigger the nodemon for you by running either `npm start` or `npm run dev ` so we don't really need to keep both ways I reckon.